developer-docs/upgrading_kubernetes.md

```diff
@@ -54,9 +54,10 @@ Once these changes are made, submit a PR for review and let CI complete. When CI
 
 Next, we need to create a release candidate (RC). The Drone (CI) process that builds the release itself can be monitored [here](https://drone-publish.rancher.io/rancher/rke2/).
 
-* Click "Releases"
-* Click "Draft new release"
-* Enter the desired version into the "Tag version" box. 
+1. Click "Releases"
```

Comment:
Let's keep the asterisks here to be consistent with the reset of the docs and the ones in different branches. We can do a full reformatting in a separate PR.